specs/src/specs/params.md

```diff
@@ -42,9 +42,9 @@ are blocked by the `x/paramfilter` module.
 | ibc.Transfer.SendEnabled | true | Enable sending tokens via IBC. | True |
 | ibc.Transfer.ReceiveEnabled | true | Enable receiving tokens via IBC. | True |
 | slashing.SignedBlocksWindow | 5000 | The range of blocks used to count for downtime. | True |
-| slashing.MinSignedPerWindow | 5 | Minumum signatures in the block. | True |
+| slashing.MinSignedPerWindow | 0.05 (5%) | Minimum proportion of blocks signed per window to avoid being slashed. | True |
```

Comment:
@cmwaters @liamsi this was wrong. 

```suggestion
| slashing.MinSignedPerWindow | 0.5 (50%) | Minimum proportion of blocks signed per window to avoid being slashed. | True |
```

Ref: https://github.com/cosmos/cosmos-sdk/blob/d6d929843bbd331b885467475bcb3050788e30ca/x/slashing/types/params.go#L18